files/lib/system/conversation/command/Leave.class.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace wcf\system\conversation\command;
+
+use wcf\data\conversation\Conversation;
+use wcf\data\conversation\ConversationList;
+use wcf\system\clipboard\ClipboardHandler;
+use wcf\system\log\modification\ConversationModificationLogHandler;
+use wcf\system\user\storage\UserStorageHandler;
+use wcf\system\WCF;
+
+/**
+ * Command for leaving a conversation.
+ *
+ * @author      Olaf Braun
+ * @copyright   2001-2025 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.2
+ */
+final class Leave
```

Comment:
I would prefer the name `LeaveConversation`. Then the naming would be in line with other commands.